Dockerfile

```diff
@@ -61,10 +61,10 @@ ENV K8S_METADATA_FILTER_BEARER_CACHE_TTL "3600"
 ENV VERIFY_SSL "true"
 
 COPY --from=builder /var/lib/gems /var/lib/gems
-
 COPY ./conf.d/ /fluentd/conf.d/
 COPY ./etc/* /fluentd/etc/
-COPY ./lib/fluent/plugin/* /fluentd/plugins/
 COPY ./entrypoint.sh /fluentd/
+COPY ./fluent-plugin-kubernetes_sumologic*.gem ./
+RUN gem install fluent-plugin-kubernetes_sumologic
```

Comment:
Should this be here or in the base image? The reason for the layer builds was to remove the build tools not needed in the final image, see RUN on line 6

Reply:
I'm not very sure about this. [The example](https://github.com/fluent/fluentd-docker-image#31-for-current-images) from fluentd doesn't use any builder image. I can give it a try

Reply:
Looks like it works. Move it to builder